src/main/java/com/synopsys/integration/detect/tool/signaturescanner/operation/CreateScanBatchOperation.java

```diff
@@ -80,6 +86,12 @@ private ScanBatch createScanBatch(
         signatureScannerOptions.getReducedPersistence()
             .ifPresent(scanJobBuilder::reducedPersistence);
 
+        // Someday the integrated matching enabled option will (we think) go away, and we'll always provide
+        // detectRunUuid as correlationId, but for now it's optional.
+        if (Boolean.TRUE.equals(signatureScannerOptions.isIntegratedMatchingEnabled())) {
```

Comment:
nitpick, maybe on opportunity to clean this up.

Reply:
Done, thanks!